CMakeLists.txt

```diff
@@ -154,7 +173,7 @@ if (WITH_CC_SERVER)
     if (MHD_FOUND)
```

Comment:
Maybe this if else can also be gotten rid of by adding a `REQUIRED` to the `find_package(MHD)`

Reply:
We will keep that structure for now.